docs/template_plugin/src/template_function_transformation.cpp

```diff
@@ -10,7 +10,7 @@ using namespace ngraph;
 
 // ! [function_pass:template_transformation_cpp]
 // template_function_transformation.cpp
-bool MyFunctionTransformation::run_on_function(std::shared_ptr<ngraph::Function> f) {
+bool pass::MyFunctionTransformation::run_on_function(std::shared_ptr<ngraph::Function> f) {
```

Comment:
std::shared_ptr&

Reply:
This is the original function signature that I do not change.